joboard_home/views.py

```diff
@@ -1,5 +1,16 @@
-from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render, redirect
+from user_notification.models import JobAlert
 
 
 def home_page(request):
     return render(request, 'joboard_home/home_page.html')
+
+
+@login_required
+def show_details(request):
```

Comment:
this is a generic name function
please change function name to be more indicative